### PR TITLE
Assignment 5 @VeraKorchemnaya @AndrewKorchemniy

### PR DESCRIPTION
**Pull request by AndrewKorchemniy**
### Deployed to: https://brave-stone-07b629c1e.3.azurestaticapps.net/

**Can edit words:**
coolbeans
P@ssw0rd

**Cannot edit words:**
uncoolbeans
P@ssw0rd

### CODE REVIEW

**Comment by StrangeRanger** [comment]
As a note, it looks like your app has a problem with the front end connecting to the backend/database

Screenshot 2023-06-06 at 9 56 53 PM

Additionally, when trying to run the Wordle.Api locally, I'm prompted with the following error:
```
/usr/local/share/dotnet/dotnet /Users/hunter/spring/Wordle.Api/Wordle.Api/bin/Debug/net7.0/Wordle.Api.dll
Unhandled exception. System.Exception: JWT configuration not specified
   at Program.<Main>$(String[] args) in /Users/hunter/spring/Wordle.Api/Wordle.Api/Program.cs:line 74

Process finished with exit code 134.
```

**Review by StrangeRanger — COMMENTED** [review]
## Features

### Create an editor for the available words ❌✅
  - Create a page named "WordEditor" that allows words to be edited ❌✅
    - Support deleting a word ❌✅
    - Support adding a new word ❌✅
    - Editing words is not supported ❌✅
    - Duplicate words should not be allowed ❌✅
    - Support changing the common word boolean flag ❌✅
    - Words should be sorted alphabetically ❌✅
  - Provide a text box for searching words AS YOU TYPE which supports searching for words starting with letters specified ❌✅
  - Provide a menu item for navigating to the WordEditor page ❌✅
    - Anyone can look at the word list ❌✅
    - Word list is paginated (10-100 per page) ❌✅
  - Any logged in user can change the common word flag ❌✅
    - There is a login page/component/popup/etc for users to sign in ❌✅
  - Only users over 21 years of age (based on birthday) with a claim of MasterOfTheUniverse can add and remove words ❌✅
  - Users can be set up at start up in the API project as a seed (apriori), there does not need to be a sign up mechanism ❌✅

## Other Notes

Unfortunately, at this point, I can't really provide a decent or accurate evaluation, as while it seems like you've covered several of the requirements, I have no way of currently testing if they function as they should, with the current state that it's in (refer to my first comment on this PR). Sorry that I can't provide any positive criticism for your project right now.

**Comment by AndrewKorchemniy** [comment]
> ## Features
> ### Create an editor for the available words ❌✅
> * Create a page named "WordEditor" that allows words to be edited ❌✅
>   
>   * Support deleting a word ❌✅
>   * Support adding a new word ❌✅
>   * Editing words is not supported ❌✅
>   * Duplicate words should not be allowed ❌✅
>   * Support changing the common word boolean flag ❌✅
>   * Words should be sorted alphabetically ❌✅
> * Provide a text box for searching words AS YOU TYPE which supports searching for words starting with letters specified ❌✅
> * Provide a menu item for navigating to the WordEditor page ❌✅
>   
>   * Anyone can look at the word list ❌✅
>   * Word list is paginated (10-100 per page) ❌✅
> * Any logged in user can change the common word flag ❌✅
>   
>   * There is a login page/component/popup/etc for users to sign in ❌✅
> * Only users over 21 years of age (based on birthday) with a claim of MasterOfTheUniverse can add and remove words ❌✅
> * Users can be set up at start up in the API project as a seed (apriori), there does not need to be a sign up mechanism ❌✅
> 
> ## Other Notes
> Unfortunately, at this point, I can't really provide a decent or accurate evaluation, as while it seems like you've covered several of the requirements, I have no way of currently testing if they function as they should, with the current state that it's in (refer to my first comment on this PR). Sorry that I can't provide any positive criticism for your project right now.

**Comment by AndrewKorchemniy** [comment]
Sorry you couldn't review it properply, looks like an oversight on my end with the backend not successfully running. I just pushed out a change that should fix it.

File: wordle-web/src/components/ActiveUser.vue [hunk 1]
```diff
@@ -1,49 +1,76 @@
...
-  <v-dialog :model-value="showDialog" @update:model-value="close" persistent>
-    <v-card>
-      <v-card-text>
-        Set or change your username:
-        <v-text-field ref="$input" v-model="newName" @keyup.enter="confirm" />
+  <v-dialog :model-value="showDialog" @update:model-value="close">
+    <v-card
+      :class="display.xs ? 'w-100' : display.sm ? 'w-75' : display.md ? 'w-50' : 'w-25'"
+      class="justify-center mx-auto"
+    >
```
Comment (AndrewKorchemniy) [inline wordle-web/src/components/ActiveUser.vue:10]: Is there a better way of making the size responsive - or is this already the best way?

Comment (meghanmae) [reply wordle-web/src/components/ActiveUser.vue:10]: This is an acceptable way to do it, however, vuetify is pretty good at authorizing things if you use percentages or max-widths. Were you running into resizing issues with your login dialog?

**Review by meghanmae — COMMENTED** [review]
Great job! I like that the UI hides/disables things depending on your access level as well.

**Comment by meghanmae** [comment]
You do seem to have a small bug in your code, took a quick look but not totally sure why it is happening. When I add a new word, (e.g. 'aaaaa') it doesn't appear in the list in alphabetical order, but it does get added. I can see the network call succeeds and I can search for it in the list of words.

**Pull request closed without merging**